python/pyspark/sql/dataframe.py

```diff
@@ -2064,6 +2064,7 @@ def toDF(self, *cols):
     @since(1.3)
     def toPandas(self):
         """
+        :param coerce_float: default False, if Ture, will handle decimal type to np.float64 instand of type object.
```

Comment:
There's no such param. Also, we don't convert it to string. Also, Arrow optimization code path should also be fixed BTW.